.travis.yml

```diff
@@ -89,4 +89,9 @@ jobs:
       script:
         - composer install
         - ./vendor/bin/phpcs -s
-          --sniffs=WordPress.WP.I18n,Generic.PHP.Syntax,WordPress.Security.EscapeOutput,WordPress.Security.NonceVerification .
\ No newline at end of file
+          --sniffs=WordPress.WP.I18n,Generic.PHP.Syntax,WordPress.Security.EscapeOutput .
+    - stage: test
```

Comment:
I'm curious as to why the nonce stuff is separate from the other sniffs?

Reply:
Because we need to only check for errors and ignore warnings for the nonce checks. The `-n` argument is the difference.

Reply:
Ah, gotcha!